_themes/conan/layout.html

```diff
@@ -158,11 +158,11 @@
           {% include "breadcrumbs.html" %}
           <!--<select id="search" name="state" multiple="true" onchange="window.location = reldir + this.value;" style="width:90%"></select>
           <hr/>-->
-          <!--<div class="admonition important">
+          <div class="admonition important">
             <p class="admonition-title">Announcement</p>
-            <p>Check out the new free <a
-            href=https://blog.conan.io/2020/09/24/New-conan-training-series.html>Conan training courses on JFrog Academy</a>!
-          </div>-->
+            <p>JFrog Bintray is being sunset. <a
+            href=https://blog.conan.io/2021/03/31/Bintray-sunset-timeline.html> Please refer this blog post for more detail.</a>
```

Comment:
Is this valid HTML?

```suggestion
            <p>JFrog Bintray is being sunset. <a
            href="https://blog.conan.io/2021/03/31/Bintray-sunset-timeline.html"> Please refer this blog post for more detail.</a></p>
```

Reply:
It seems so: https://html.spec.whatwg.org/multipage/grouping-content.html#the-p-element